examples/fingerprint_picturetest.py

```diff
@@ -0,0 +1,250 @@
+# SPDX-FileCopyrightText: 2023 Jim McKeown
+# SPDX-License-Identifier: MIT
+
+import time
+import numpy as np
+from matplotlib import pyplot as plt
+import serial
+import adafruit_fingerprint
+
+"""Added 'View Print' and 'Preview and Find Print' to fingerprint_simpletest.py"""
+"""This example only works on single board computers with the use of Blinka"""
```

Comment:
Rather than note the difference from `fingerprint_simpletest.py`, let's ouright say what this example will do so users will immediately know :)

Also, I would move this above the import statements and use only a single set of triple quotes, like this:

```python
# SPDX-FileCopyrightText: 2023 Jim McKeown
# SPDX-License-Identifier: MIT

"""
This is the module docstring!
"""

import time
...
```